styles/index.css

```diff
@@ -2,28 +2,45 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-
-  padding: 60px 20px;
-  gap: 90px;
-  max-width: 800px;
+  background-color: white;
+  padding: 20px 48px;
+  gap: 70px;
+  max-width: 1500;
```

Comment:
1500px the code editor won't complain on this but the browser can't interpret raw numerical value

Reply:
yes, I will add px